frontend/components/CreateMatchForm.js

```diff
@@ -0,0 +1,215 @@
+import React, {useEffect, useState} from 'react';
+import FormHeader from "./util/FormHeader";
+import {Button, Center, Flex, FormControl, Text, VStack} from "native-base";
+import Api from "./util/Api";
+import {FontAwesome} from "@expo/vector-icons";
+import DateTimePicker from "@react-native-community/datetimepicker"
+import SearchableDropdown from "react-native-searchable-dropdown";
+import {Platform, StyleSheet} from "react-native";
+
+const CreateMatchForm = () => {
+    const [firstPlayer, setFirstPlayer] = useState({})
+    const [secondPlayer, setSecondPlayer] = useState({})
+
+    const [players, setPlayers] = useState([])
+
+    const [date, setDate] = useState(new Date())
+    const [mode , setMode] = useState('date')
+    const [show, setShow] = useState(false)
+    const [showSearch1, setShowSearch1] = useState(true)
+    const [showSearch2, setShowSearch2] = useState(true)
+
+    const [dateString, setDateString] = useState(date.getDate() + "-" + date.getMonth() + "-" + date.getFullYear())
+    const [timeString, setTimeString] = useState(date.getMinutes() < 10 ? date.getHours() + ":0" + date.getMinutes() : date.getHours() + ":" + date.getMinutes())
+
+    const getAllPlayers = () => {
+        Api.get('/players' // TODO type proper url when backend is ready
+        ).then(res => {
+            setPlayers(res.data)
+        }).catch(e => {
+            console.log(e) // TODO handle it somehow
+        })
+    }
+
+    useEffect(() => {
+        console.log(typeof firstPlayer.name)
+        getAllPlayers()
+    }, [])
+
+    const showMode = (mode) => {
+        setShow(true)
+        setMode(mode)
+    }
+
+    const onChange = (event, selectedDate) => {
+        event.preventDefault()
+        const currDate = selectedDate || date
+        setShow(Platform.OS === "ios")
+        setDateString(date.getDate() + "-" + date.getMonth() + "-" + date.getFullYear()) // example date format
+        setTimeString(date.getMinutes() < 10 ? date.getHours() + ":0" + date.getMinutes() : date.getHours() + ":" + date.getMinutes()) // example time format
```

Comment:
Same as above one. Maybe showing hours on two places with leading '0' is good idea too?
```suggestion
        setTimeString(date.getHours() + ":" + date.getMinutes().toString().padStart(2, "0"))
```